portfolio/src/main/webapp/requests/queries/index.js

```diff
@@ -0,0 +1,24 @@
+const get_comments = async () => {
```

Comment:
nit: the Google Javascript Style Guide [suggests](https://google.github.io/styleguide/jsguide.html#naming-method-names) that method names should be lowerCamelCased, i.e. `getComments`